### PR TITLE
Add note on rsync

### LINKED ISSUE #100: Mention rsync for file transfer
I think it is worth mentioning `rsync` as an alternative to `scp` and to include some basic information about it on an Extras page (see also #27). It is arguably too advanced for an intro course, given its many options, but as new users become more experienced with HPC they will likely find `rsync` to be a better tool for transferring files. The basic syntax is similar to `scp`, but It is typically faster and easier to use when transferring large and/or many files and creating backups.

### PR DESCRIPTION
**Pull request by drkrynstrng**
Closes #100.

### CODE REVIEW

File: _episodes/15-transferring-files.md [hunk 1]
```diff
@@ -167,6 +167,14 @@ to transfer files.
 > files.
 {: .callout}
 
+> ## A note on rsync
```
Comment (psteinb) [inline _episodes/15-transferring-files.md]: I suggest to put this call-out at [the end](https://github.com/hpc-carpentry/hpc-intro/blob/a41c3558f3e0716630b1f29993863c3dcf2faa1a/_episodes/15-transferring-files.md#transferring-single-files-and-folders-with-scp) of the `scp` section.

File: _episodes/15-transferring-files.md [hunk 2]
```diff
@@ -167,6 +167,14 @@ to transfer files.
...
+>
+> As you gain experience with transferring files, you may find the above methods limiting. The
+> [rsync](https://rsync.samba.org/) utility provides advanced features for file transfer and is
+> typically faster. It is especially useful for transferring large and/or many files and creating
+> synced backup folders.
```
Comment (psteinb) [inline _episodes/15-transferring-files.md]: Please add examples to be on par with scp. I suggest to use the single file transfer example and the folder based example.

**Comment by psteinb** [comment]
Thanks for sending the PR. feel free to close #100.

File: _episodes/15-transferring-files.md (hunk 1 above)
Comment (psteinb) [reply _episodes/15-transferring-files.md]: any opinions?

File: _episodes/15-transferring-files.md (hunk 2 above)
Comment (psteinb) [reply _episodes/15-transferring-files.md]: To make myself concrete, please add one example of a download operation. After that, this PR is ready for merge from my point of view.

Comment (drkrynstrng) [reply _episodes/15-transferring-files.md]: Okay, I've added a download example. It should be merge-able, but let me know if I need to update the branch to the most recent commits.

**Pull request merged**